tests/test_migrate.py

```
@@ -0,0 +1,1417 @@
+        (logs_dir / "app.log").write_text("log content")
+        new_home = tmp_path / "home"
+        new_home.mkdir()
+
+        result = migrate_logs([legacy_root], new_home, move=True)
+
+        assert result["success"] is True
+        assert result["action"] == "move"
+        assert (new_home / "logs").exists()
+        assert not (legacy_root / "logs" / "app.log").exists()
+
+    def test_timestamped_log_names(self, tmp_path, monkeypatch):
```

Ruff (0.14.14)

[warning] Unused method argument: `monkeypatch`

(ARG002)